### PR TITLE
remove kit id from account signup

### PR DESCRIPTION
**Pull request by raginirai553**

### CODE REVIEW

File: microsetta_interface/tests/test_implementation.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def __init__(self, status_code, text, json_val, headers=None):
 
     def test_authrocket_callback_noargs_error(self):
         # We are observing exceptions on /authrocket_callback which occur with
-        # malformed requests. Currently, we are throwing a 500, but instead
+        # malformed requests. Currently,we are throwing a 500, but instead
```
Comment (cassidysymons) [inline microsetta_interface/tests/test_implementation.py:70]: Can you explain why you removed the space from this comment?

File: microsetta_interface/implementation.py [hunk 2]
```diff
@@ -53,6 +53,7 @@ class Source:
 ADMIN_MODE_KEY = 'admin_mode'
 LOGIN_INFO_KEY = 'login_info'
 LANG_KEY = "language"
+claim_kit_name_hint = None
```
Comment (cassidysymons) [inline microsetta_interface/implementation.py:56]: This is only used in the scope of the get_source function, please move it into that scope.

**Review by cassidysymons — CHANGES_REQUESTED** [review]

File: microsetta_interface/implementation.py (hunk 2 above)
Comment (raginirai553) [reply microsetta_interface/implementation.py:56]: sure doing

File: microsetta_interface/tests/test_implementation.py (hunk 1 above)
Comment (raginirai553) [reply microsetta_interface/tests/test_implementation.py:70]: no reason, it must be a typo. It is not a technically thought change

**Pull request closed without merging**